ui/page/privacy/manual_mixer_setup_page.go

```diff
@@ -122,65 +116,110 @@ func (pg *ManualMixerSetupPage) OnNavigatedTo() {
 // to be eventually drawn on screen.
 // Part of the load.Page interface.
 func (pg *ManualMixerSetupPage) Layout(gtx layout.Context) layout.Dimensions {
-	body := func(gtx C) D {
-		page := components.SubPage{
-			Load:       pg.Load,
-			Title:      values.String(values.StrManualSetUp),
-			BackButton: pg.backButton,
-			Back: func() {
-				pg.ParentNavigator().CloseCurrentPage()
-			},
-			Body: func(gtx C) D {
-				return pg.Theme.Card().Layout(gtx, func(gtx C) D {
-					gtx.Constraints.Min.X = gtx.Constraints.Max.X
-					return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
-						layout.Flexed(1, func(gtx C) D {
-							return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
+	return layout.Inset{Top: values.MarginPadding40}.Layout(gtx, func(gtx C) D {
+		return pg.Theme.Card().Layout(gtx, func(gtx C) D {
+			gtx.Constraints.Min.X = gtx.Constraints.Max.X
+			return layout.Flex{Axis: layout.Vertical, Alignment: layout.Start}.Layout(gtx,
+				layout.Rigid(func(gtx C) D {
+					return layout.Center.Layout(gtx, func(gtx C) D {
+						return layout.Inset{Top: values.MarginPadding25}.Layout(gtx, func(gtx C) D {
+							return layout.Flex{Axis: layout.Vertical, Alignment: layout.Start}.Layout(gtx,
 								layout.Rigid(func(gtx C) D {
-									return pg.mixerAccountSections(gtx, values.String(values.StrMixedAccount), func(gtx layout.Context) layout.Dimensions {
-										return pg.mixedAccountSelector.Layout(pg.ParentWindow(), gtx)
-									})
+									return layout.Flex{Alignment: layout.Start}.Layout(gtx,
+										layout.Rigid(func(gtx C) D {
+											return pg.backFromManual.Layout(gtx)
+										}),
+										layout.Rigid(func(gtx C) D {
+											label := pg.Theme.H6(values.String(values.StrSetUpStakeShuffleAutoOrManualG))
+											return layout.Inset{
+												Top:  values.MarginPadding10,
+												Left: values.MarginPadding5,
+											}.Layout(gtx, label.Layout)
+										}),
+										// TODO: The Label above will not go to the left unless I eat up all the space
+										// on the right here. This cannot be the correct way to implement this. Find a
+										// way to remove this spacer.
+										layout.Rigid(func(gtx C) D {
+											return layout.Spacer{Width: 100000000}.Layout(gtx)
+										}),
```

Comment:
Can't seem how to anchor this to the left. Nested flexed layouts don't seem to work as I would expect, although it's more likely I am implementing them wrong. Here and in another place.

Reply:
I think wrapping in a `layout.E.Layout` will do the job. I saw where `layout.Center.Layout` was used to centralize a widget inside a flexed child.

Reply:
I figured it out! Needed to set `gtx.Constraints.Min.X = gtx.Constraints.Max.X` in the parent layout so it could go left.